packages/components/tsconfig.json

```diff
@@ -6,7 +6,8 @@
 			"gutenberg-env",
 			"gutenberg-test-env",
 			"jest",
-			"@testing-library/jest-dom"
+			"@testing-library/jest-dom",
```

Comment:
Its weird that this suddenly needs to be added to allow the builds to be built?

Reply:
This problem was always there looks like. When I installed a new types package it raised its head. So I added node here and in the dataviews package to fix it.

It might be worth adding it to the base tsconfig but I'll leave that for now.